sdk/tests/conformance/reading/read-pixels-test.html

```diff
@@ -39,6 +39,18 @@
   gl = wtu.create3DContext(canvas, {antialias: antialias});
   var contextVersion = wtu.getDefault3DContextVersion();
 
+  debug("");
+  debug("Test null receiver");
+  gl.readPixels(0, 0, 1, 1, gl.RGBA, gl.UNSIGNED_BYTE, null);
+  wtu.glErrorShouldBe(gl, gl.INVALID_VALUE);
```

Comment:
There's a related test that could be added in a commit that tests the readPixels error values:

https://registry.khronos.org/webgl/specs/latest/2.0/#3.7.10

> undefined readPixels(GLint x, GLint y, GLsizei width, GLsizei height, GLenum format, GLenum type, [AllowShared] ArrayBufferView dstData, GLuint dstOffset) 

...

> If dstData doesn't have enough space for the read operation starting at dstOffset, generate INVALID_OPERATION.

Reply:
Done.